django_feature_flip/models.py

```diff
@@ -2,7 +2,11 @@
 
 
 class Feature(models.Model):
-    name = models.CharField(max_length=40, unique=True)
+    name = models.CharField(max_length=40, unique=True, null=False)
+    totally_enabled = models.BooleanField(default=False, null=False)
```

Comment:
Null is false by default, no need to add it. Just a comment, no need to change it.